.github/workflows/ci.yml

```diff
@@ -14,10 +14,29 @@ on:
 jobs:
 
   spec:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby_version:
+          - '2.5'
+          - '2.7'
```

Comment:
Perhaps out of scope, but any plans to test on Ruby 3.0 & 3.1? 3.1 is what I use, so I'd expect it to work but it'd be nice to verify it.

For the changelog alone I would submit it as a separate PR.